backend/src/db-utils.js

```diff
@@ -84,14 +85,14 @@ const queryOnlineNodes = async () => {
   ]);
 };
 
-//new query for new dashboard
+// query for new dashboard
 const queryDashboardBlocksStats = async (options) => {
   async function queryLatestBlockHeight({ dataSource }) {
     let query;
     if (dataSource === DS_INDEXER_BACKEND) {
       query = `SELECT block_height FROM blocks ORDER BY block_height DESC LIMIT 1`;
     } else {
-      query = `SELECT height AS block_height FROM blocks ORDER BY height DESC LIMIT 1`;
+      query = `SELECT height as block_height FROM blocks ORDER BY height DESC LIMIT 1`;
```

Comment:
`AS` is an SQL keyword, so given we opt into using uppercase for all the SQL keywords, let's keep `AS` in uppercase, please.